### PR TITLE
Move transcripts

### PR DESCRIPTION
**Pull request by stephencelis**
So #13 turns out to be more finicky in use. Let's try this again by using submodules (for private), and `make` (for OSS).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
-	make xcodeproj
+# private
 
-untranscripts:
-	echo > .git/info/exclude
-	git update-index --no-assume-unchanged Sources/pointfreeco/Transcripts/AllEpisodes.swift
-	git checkout -- Sources/pointfreeco/Transcripts/
-	git clean --force Sources/pointfreeco/Transcripts/
-	make xcodeproj
+start:
```
Comment (stephencelis) [inline Makefile:31]: We can make our private bootstrap process better in the future, but for now I'm prioritizing OSS.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (stephencelis) [inline .env.example:1]: Added an example `.env` file (that gets copied over for `make`/`make oss`) for easier OSS configuration.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,52 @@
...
+  - Download and bootstrap:
+    ``` sh
+    git clone https://github.com/pointfreeco/pointfreeco-server.git
+    cd pointfreeco-server
+    make
+    ```
+
+  - Open http://0.0.0.0:8080 in your browser.
+
+Now that you're (hopefully) up and running, you can edit the `.env` file and
```
Comment (stephencelis) [inline README.md:41]: We have `AppEnvironment`, so I wonder if we wanna bootstrap mocks in the future, too. For now, config instructions for the brave few.

**Comment by stephencelis** [comment]
So this works nicely! I think we could make it a bit more of a universal solution and rather than lock it to:

```
Transcripts.example/ -> Sources/pointfreeco/Transcripts/
```

We could have the private repo responsible for everything from including transcripts to bootstrapping the app. So the default, in `Setup.example/`, could mock our dependencies by default (allowing for things like auth and subscriptions to work without having to set up a GitHub application and Stripe account.

**Comment by stephencelis** [comment]
(If ya wanna bikeshed `Setup.example/` to `Bootstrap.example/` or anything else, I'm all ears!)

**Review by mbrandonw — APPROVED** [review]
much better!

**Pull request merged**